september/src/main/java/edu/asu/diging/gilesecosystem/september/core/config/SecurityContext.java

```diff
@@ -39,7 +39,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 // Anyone can access the urls
                 .antMatchers("/", "/resources/**", "/login/authenticate",
-                        "/logout").permitAll()
+                        "/logout", "/datatable").permitAll()
```

Comment:
so basically you make the messages available to anyone? why would a user have to login then? 

the endpoint should fit the page it is used on (under 'admin' I assume) and only be accessible by admins.